client/src/pages/Login/useInput.ts

```diff
@@ -0,0 +1,11 @@
+import { ChangeEvent, useCallback, useState } from "react";
+
+type UseInputReturn = [string, (ev: ChangeEvent<HTMLInputElement>) => void];
+
+export const useInput = (initValue: string): UseInputReturn => {
+  const [state, setState] = useState(initValue);
+  const onChange = useCallback((ev: ChangeEvent<HTMLInputElement>) => {
+    setState(ev.target.value);
+  }, []);
+  return [state, onChange];
```

Comment:
hook not needed for such things

Reply:
UseInput is used many times so to prevent duplications, I've extracted it into a hook